src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -368,10 +352,24 @@ class SourcesTree extends Component<Props, State> {
   }
 }
 
+function getSourceForTree(
+  state: SourcesState,
```

Comment:
small nit, but i think it is nicer to describe it as AppState -> state: AppState,